.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+### Checklist
+
+- [ ] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
```

Comment:
We should add a checklist item that says something like "Any modified or new methods or classes have helpful Javadoc and code is thoroughly commented." Maybe we can link to a commenting style that we like.

Reply:
Yeah a style guide would definitely be helpful. I'll just add a note about javadoc for now.